imgui/core.pyx

```diff
@@ -1374,6 +1393,307 @@ def is_window_collapsed():
+
+    .. visual-example::
+        :auto_layout:
+        :height: 200
+        :width: 200
+        :click: 80 40
+
+        selected = [False, False]
+        imgui.begin("Example: selectable")
+        _, selected[0] = imgui.selectable(
+            "1. I am selectable", selected[0]
+        )
+        _, selected[1] = imgui.selectable(
+            "2. I am selectable too", selected[1]
+        )
+        imgui.text("3. I am not selectable")
+        imgui.end()
+
+    Args:
+        label (str): The label.
+        selected (bool): defines if item is selected or not.
+        flags: Selectable flags. See:
+            :ref:`list of available flags <selectable-flag-options>`.
+        width (float): button width.
+        height (float): button height.
+
+    Returns:
+        tuple: a ``(opened, selected)`` two-tuple indicating if item was
+        clicked by the user and the current state of item.
+
+    .. wraps::
+        bool Selectable(
+            const char* label,
+            bool selected = false,
+            ImGuiSelectableFlags flags = 0,
+            const ImVec2& size = ImVec2(0,0)
+        )
+
+        bool Selectable(
+            const char* label,
+            bool* selected,
+            ImGuiSelectableFlags flags = 0,
+            const ImVec2& size = ImVec2(0,0)
+        )
+    """
+    cdef cimgui.bool inout_selected = selected
+    return cimgui.Selectable(
+        _bytes(label),
+        &inout_selected,
+        flags,
+        _cast_args_ImVec2(width, height)), inout_selected
+
+
+def listbox(
+    str label,
+    int current,
+    list items,
+    int items_count,
+    int height_in_items=-1
+):
+    """Show listbox widget.
+
+    .. visual-example::
+        :auto_layout:
+        :height: 100
+
+        current = 2
+        imgui.begin("Example: listbox widget")
+
+        clicked, current = imgui.listbox(
+            "List", current, ["first", "second", "third"], 3
+        )
+
+        imgui.end()
+
+    Args:
+        label (str): The label.
+        current (int): index of selected item.
+        items (list): list of string labels for items.
+        items_count (int): items count
+        height_in_items (int): height of dropdown in items. Defaults to -1
+            (autosized).
+
+    Returns:
+        tuple: a ``(changed, current)`` tuple indicating change of selection
+        and current index of selected item.
+
+    .. wraps::
+        bool ListBox(
+            const char* label,
+            int* current_item,
+            const char** items,
+            int items_count,
+            int height_in_items = -1
+
+    """
+    cdef int inout_current = current
+
+    cdef const char** in_items = <const char**> malloc(len(items) * sizeof(char*))
+    for i in range(len(items)):
+        in_items[i] = strdup(_bytes(items[i]))
+
+    opened = cimgui.ListBox(
+        _bytes(label),
+        &inout_current,
+        in_items,
+        items_count,
+        height_in_items)
+
+    free(in_items)
```

Comment:
There is a memory issue. You have freed `in_items` array but have not freed each string created by `strdup` separately.